### DISCUSSION TITLE
Wrong Frequency Displayed

### DISCUSSION THREAD

**Discussion by pmaine** [AlexandreRouma/SDRPlusPlus#371] [Q&A]
I am using a HackRF SDR in Ubuntu. Please see the attachment. The displayed frequency is 142.400.000 however I am hearing NOA Weather that ![image](https://user-images.githubusercontent.com/5252230/134262235-197902ae-7f5b-4619-8252-240af113fff7.jpeg)is actually on 162.400.000.

What's the problem?

**Comment by AlexandreRouma** [comment]
this is just an image of the signal, the hackrf is know for this kind of stuff
It's not dispalying the "wrong" frequency, it's just that you've got a strong signal leaking into the passband
